_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,174 @@
+# Lecture 8 : Causal Discovery and Inference
+Causation implies correlation(or dependence) but correlation doesn't imply causation. Consider the case of chocolate consumption in a country compared with the number of nobel laureates in the country
+
+<figure>
+  <div class="row">
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe1.jpg' | relative_url }}"  width="600" height="400" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>Chocolate consumptions vs Nobel Laureate</strong>
+  </figcaption>
+</figure>
+
+From this plot, we can see that they are highly correlated. But it is absurd to say that if we want to increase the number of nobel laureates in a country, we need to eat more chocolate. This is where the idea of causality comes in. They are correlated but chocolate consumption is not the cause of nobel laureates. The following equations capture this difference mathematically
+
+X and Y are dependent if and only if $$\exists x_1 \neq x_2$$ such that:
+
+$$P(Y|X=x_{1}) \neq P(Y|X=x_2) $$
+
+X is a cause of Y if and only if $$ \exists x_1 \neq x_2$$ such that:
+
+$$  P(Y|do(X=x_1) \neq P(Y|X=x_2) $$
+
+The definition requires just a pair of distinct $$ x_1, x_2 $$ to have different conditional distribution of Y. Y might have same conditional distribution over a range of x values. But if we find even a single pair for which the conditional distribution is different, X and Y are dependent. Same goes for causation 
+
+The causation definition is circular in the sense that if we don't know the causal relation of the other variables, we can't define the intervention of X and thus can't find the causal relation of X and Y. So to define one causal relation, we need all other causal relations. 
+
+## Causal Thinking
+### Simpson's Paradox
+Let's say there are two type of patients, one with small kidney stones and other with large kidney stones. And there are two types of treatments to choose from, A and B. From the data we can see that, if we consider the patients separately, in both the cases treatment A is better due to higher recovery rate. But if we combine both the categories of patients, we see that the treatment B has higher recovery rate. So, as a doctor, what treatment would you suggest to a patient.
+
+<figure>
```

Comment:
I think the figure looks better in the web browser if the two images are placed side-by-side

Reply:
Thanks @alshedivat for the fix